syntax = "proto3";

package order;

option go_package = "grpc-go-client/order";

service OrderService {
  rpc CreateOrder(OrderRequest) returns (OrderResponse);
}

message OrderRequest {
  int32 order_id = 1;
  string item = 2;
  int32 quantity = 3;
}

message OrderResponse {
  string status = 1;
  string message = 2;
}
